pages/learn/elective.tsx

```diff
@@ -244,8 +245,7 @@ export const getServerSideProps: GetServerSideProps = async ({
     }),
 
     // Check if the time now is in an Enrollment Period.
-    // await mysk.fetch<boolean>("/v1/subjects/electives/in-enrollment-period"),
-    { data: true },
+    await mysk.fetch<boolean>("/v1/subjects/electives/in-enrollment-period"),
```

Comment:
Adding a check for the Enrollment Period would make it possibly more prone-free to mis-enrollment!

(As I understand the comment here...)